src/platform_impl/macos/view.rs

```diff
@@ -489,79 +482,6 @@ extern "C" fn first_rect_for_character_range(
     }
 }
 
-extern "C" fn insert_text(this: &Object, _sel: Sel, string: id, _replacement_range: NSRange) {
-    trace!("Triggered `insertText`");
-    unsafe {
-        let state_ptr: *mut c_void = *this.get_ivar("winitState");
-        let state = &mut *(state_ptr as *mut ViewState);
```

Comment:
I'm not sure about this change tbh. However I can't prove it myself that this thing is requited. So I need a time to go through the history to understand what this thing is about. I'd suggest you to do the same, since there should be a reason for it.

Reply:
I've been going through old issues, and seeing that I can't reproduce any of them. I'll keep digging.